### ISSUE TITLE
Rename repository to sourcejs-core to make it more findable and 'easier' to fork

### ISSUE THREAD

**Issue by ndelangen** [sourcejs/Source#216]
Searching github for sourcejs does not yield what you'd expect. I'd expect sourcejs/Source to be the top hit, but even when sorting by number of stars or forks, the Source repository does not appear at all.

I suggest renaming the repository to sourcejs-core.

A nice side-effect will be that when someone forks sourcejs-core the repository name will still be linked to sourcejs.

**Comment by operatino** [comment]
Hm, that's a valid point, let's also see how other organizations with multiple repos solve that.

**Comment by ndelangen** [comment]
Gulp uses `gulpjs/gulp`
https://github.com/search?utf8=%E2%9C%93&q=gulp

So does Grunt:
https://github.com/search?utf8=%E2%9C%93&q=grunt&type=Repositories&ref=searchresults

AngularJS has `angular/Angular`
https://github.com/search?utf8=%E2%9C%93&q=angular&type=Repositories&ref=searchresults

react has `facebook/react`
https://github.com/search?utf8=%E2%9C%93&q=react&type=Repositories&ref=searchresults

**Comment by ndelangen** [comment]
Looks like github favors repository names & repository descriptions for searching 'tags' in.

Searching for React**JS** yields this:
https://github.com/search?o=desc&q=reactjs&ref=searchresults&s=stars&type=Repositories&utf8=%E2%9C%93

**Comment by operatino** [comment]
So probably we just need to name core repo as a main package `sourcejs`?

**Comment by ndelangen** [comment]
Yeah, sounds like that has become common practice, and it also corresponds with the npm package name.

**Comment by ndelangen** [comment]
I assume renaming it will cause githib to redirect any incoming requests to the old repository to the new url?

**Comment by operatino** [comment]
It should, but not sure how reliable this is. Will wait until 0.6.0 release, where official distribution will be handled through npm.
